web/backend/src/Server.ts

```diff
@@ -154,49 +185,58 @@ app.post('/api/logout', (req, res) => {
 // As the user is logged on the app via this express but must also be logged in
 // the react. This endpoint serves to send to the client (actually to react)
 // the information of the current user.
```

Comment:
The comment is for the statement bellow, not the `setMapAuthorization` function, which must also have its own comment describing what it does.

Reply:
This function is hard to understand without any explanation. You should at least describe what contains `list: string[][])`.

Reply:
I added some comments and while doing it I noticed that the iteration on j isn't necessary.